roles/disaster_recovery/README.md

```diff
@@ -0,0 +1,93 @@
+oVirt Disaster Recovery
+=========
+
+The `oVirt.disaster-recovery` role responsible to manage the disaster recovery scenarios in oVirt.
```

Comment:
Rename the `oVirt.disaster-recovery` to  `disaster_recovery`

Reply:
Thanks for this and other comments.
I wanted to address the renaming in the follow-up patches.
Deliberately wanted this patch to be a "clean" copy of the original :)

Reply:
Leaving ```-``` in role name would break collections, so I'd prefer to have single commit to import current status and then additional commit to fix issues. But all commits need to be within a single PR

Reply:
Yes, I rephrase myself :)
I meant follow-up **commits** (not patches). Commits inside **this patch**.
All I wanted was to have the 1st commit for just copying DR project logic (something that I already see didn't work for 100% since the CI failed). And the next ones to contain all the modifications needed to support collections.
For clear separation of all the changes done due to collections.

Reply:
> oVirt.disaster-recovery

Done.